Customer Support Procedures/Neo4j-v5.0-migration.md

```diff
@@ -0,0 +1,47 @@
+<!-- omit in toc -->
+# Neo4j migration to v5 guidelines
+
+If you are planning to migrate your Neo4j database from an older version to v5,
+this document will provide you extra details of what you need to take care of in your Linkurious Enterprise project.
+
+<!-- omit in toc -->
+# Table of contents
+- [Pre-requisites and limitations](#pre-requisites-and-limitations)
+- [Issues with cypher queries](#issues-with-cypher-queries)
+- [Migrating databases](#migrating-databases)
+
+# Pre-requisites and limitations
```

Comment:
I also hear that possibly Noe4j v5 changed something in the cluster architecture. Is it only going to impact the incremental indexing or are there other things to be aware for customers having a cluster configuration?

Reply:
You're right, the cluster architecture is changed in Neo4j v5. As explained here, incremental indexing does not work on a Neo4j 5.2.0 cluster. We expect that to be fixed in a later version, but I do not have a date yet. Otherwise, I don't think there is any other specific impact for our customers. They'll have to migrate their configuration, but that's not specific to LKE.